### PR TITLE
Netsil 1.0.0

### PR DESCRIPTION
**Pull request by luhkevin**
An update to the Netsil version -- bumping to 1.0.0!

### CODE REVIEW

**Comment by luhkevin** [comment]
When can this be merged in?

File: repo/packages/N/netsil/0/config.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
           "type": "integer"
```
Comment (ryadav88) [inline repo/packages/N/netsil/0/config.json]: Looks like there are a lot of changes over the previous package. Please create another package version 
repo/packages/N/netsil/1

**Comment by luhkevin** [comment]
Hey there, I created another package version. 
And yeah, I tested this package on DCOS with the defaults, following the instructions here: https://github.com/mesosphere/universe#universe-server

**Comment by ryadav88** [comment]
@luhkevin Can you ping me on the DC/OS community slack? I'm ravi.mesosphere there.

**Pull request merged**